java/integTest/src/test/java/glide/TransactionTestUtilities.java

```diff
@@ -98,6 +99,10 @@ public static BaseTransaction<?> transactionTest(BaseTransaction<?> baseTransact
         baseTransaction.zpopmin(key8);
         baseTransaction.zpopmax(key8);
 
+        baseTransaction
+                .zadd(zsetKey2, Map.of("one", 1.0, "two", 2.0, "three", 3.0))
+                .bzpopmax(new String[] {zsetKey2}, .1);
```

Comment:
These timeouts won't be an issue right because the set is already full? This could have been an issue if the set was initially empty and would block until it filled up? Just curious thanks!

Reply:
Yes, otherwise it blocks the transaction forever. Luckly, we have test timeout to avoid entire CI block.